Client/Assets/Search/SearchPlugins/list.json

```diff
@@ -2,7 +2,890 @@
   "default": {
     "searchDefault": "ecosia",
     "visibleDefaultEngines": [
-      "ecosia"
+        "ecosia", "wikipedia", "google-b-m", "bing", "ddg"
     ]
+  },
+  "regionOverrides": {
+    "US": {
+      "google-b-m": "google-b-1-m"
+    }
+  },
+  "locales": {
+    "ach": {
+      "default": {
+        "visibleDefaultEngines": [
+          "google-b-m", "bing", "wikipedia"
+        ]
+      }
+    },
+    "an": {
+      "default": {
+        "visibleDefaultEngines": [
+          "google-b-m", "bing", "wikipedia-an"
+        ]
+      }
+    },
+    "ar": {
```

Comment:
just wondering if we need all this? I guess it's mostly restoring the values from FF, isn't it? My question would be that here in `ar` is `visibleDefaultEngines` taken from the country or from the default?

Reply:
Good question!
I know there were a couple of ways to tackle this change and make the search engines appear as we wanted.
After a careful examination, I preferred to choose the way that tackled the choice right from their source.
There are some expectations to make for example the Google search engine works as expected in every country (selecting its own search engine) and I didn't want to apply any change that would undermine our current expectation for search engines being shown against each country.
`visibleDefaultEngines` if specified for a country, overrides the `visibleDefaultEngines` at the top. Having said that, leaving only the expected ones for each country may seem like a lot of work at the beginning, but it guarantees that all expected search engines are visible. Modifying the codebase to filter only the selected one couldn't prove a bulletproof implementation considering all the country-specific ones.
If we'd go for expanding this experiment the search engines shown will match the expected as filtedered at the source.

Reply:
@d4r1091 I understand. But this would mean that we see different search short cuts in different countries, right? 🤔 I need to think about this for a bit.

I guess for the experiment that's good enough as it does not affect our core markets (DE, FR, US, UK)

Reply:
Browsing trough the list, I can that most of the times the country specific defaults match our expectations and Google, Bing and Wikipedia are present. So for the experiment it's ok. In case we want to fully include the feature, I think we'd need to reassess to understand all the implications.

Reply:
@ecotopian 💯 , the different search shortcuts will be the ones matching what FF exposed to us, except for the search engines _we don't want_ e.g. everything but Google, Bing, DDG, and Wikipedia